packages/dashboard-frontend/src/services/workspace-client/helpers.ts

```diff
@@ -153,7 +153,7 @@ export async function getCustomEditor(
       if (cheEditorYaml.registryUrl) {
         repositoryEditorYamlUrl = `${cheEditorYaml.registryUrl}/plugins/${cheEditorYaml.id}/devfile.yaml`;
       } else {
-        repositoryEditorYamlUrl = `${pluginRegistryUrl}/plugins/${cheEditorYaml.id}/devfile.yaml`;
+        repositoryEditorYamlUrl = cheEditorYaml.id;
```

Comment:
I think `repositoryEditorYamlUrl` is not a correct variable name anymore.
` editorDefinition` sounds much better.
@svor  WDYT?

Reply:
renamed to **editorReference**